static/js/popovers.js

```diff
@@ -547,6 +547,14 @@ exports.register_click_handlers = function () {
         e.stopPropagation();
     });
 
+    $('body').on('click', '.mark_stream_as_read', function (e) {
+        var stream = $(e.currentTarget).parents('ul').attr('data-name');
+        //var topic = $(e.currentTarget).attr('data-topic-name');
```

Comment:
I assume this line is a holdover from when you were thinking about doing topics too?  I think you should -- probably would be possible to do it with very little extra code...